tests/common.py

```diff
@@ -32,6 +32,9 @@ def setup_gpus_config(section_path="serve", gpus=None, tps=None, vllm_args=lambd
     return _CFG_FILE_NAME
 
 
+@mock.patch(
```

Comment:
I am guessing this check is causing a lot of tests to make it to the previous failure point of failed to determine backend?

If that's the case, I think it would be better to change those tests to adjust to the new error.

Reply:
Turns out nothing was forcing a failed to determine backend failure.  It was always being mocked.  So this change is appropriate.